### ISSUE TITLE
Links in Shownotes mit 'target="_blank" rel="noopener"'

### ISSUE THREAD

**Issue by mabe-at** [aua-uff-code/aua-uff-co.de#4]
Gibt es irgendwo die Möglichkeit Links (zumindest in den Shownotes) mit 'target="_blank" rel="noopener"' zu versehen?

Es scheint so, als ob das in Octopod gemacht werden muss.

LG Matthias

**Comment by haslinger** [comment]
Ja könntest Du, beliebiges valides HTML ist auch valides Markdown. Dafür braucht es keine  Funktionalität in Jekyll-Octopod.

Ich würde aber davon abraten, da Du nicht weißt, was die HTML-Views der Podcatcher zulassen und was nicht, daher Links dann funktionslos sein könnten. 
Ich traue unseren Hörer:innen zu, den Back Button zu finden und zu klicken :-)

**Comment by mabe-at** [comment]
Ja, eh. Der Player merkt sich nur leider nicht, an welcher Stelle man war.
Cool wäre, wenn Octopod das schon richtig macht.

**Comment by mabe-at** [comment]
Habe gerade gesehen, dass man eigene Filter registrieren kann. Ich spiel' mich einmal damit.
